site/en/blog/chromium-chronicle-18/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "The Chromium Chronicle #18: Chromium Code Coverage"
+description: >
+  Test coverage measures how much source code is executed when a particular
+  test suite runs.
+layout: 'layouts/blog-post.njk'
+date: 2021-02-22
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hgu6uTktp2ipmuODZZhP.jpg'
+alt: >
+  Chromium Chronicle image
+tags:
+  - chromium-chronicle
+---
+
+**Episode 18:** by Veenita Joshi and Prakhar Asthana in
+Mountain View, CA (February 2021)<br>
+[Previous episodes](/tags/chromium-chronicle/)
+
+**Test coverage measures how much source code is executed** when a particular
+test suite runs. Generating coverage data has three major steps: compiling
+with instrumentation, gathering data from running instrumented tests, and
+post-processing the data. The coverage team has tools to automate this process.
+
+The [Coverage script][1] **can generate code coverage reports in just one command**.
```

Comment:
Can we please add "without waiting for Gerrit UI to be updated (Episode 3)" to this sentence? And could Episode 3 hyperlink to this older Chromium Chronicle episode: https://developer.chrome.com/blog/chromium-chronicle-3/

Reply:
done